code/game/objects/items/cards_ids.dm

```diff
@@ -210,7 +209,8 @@
 		to_chat(user, "<span class='notice'>You withdraw [amount_to_remove] credits into a holochip.</span>")
 		return
 	else
-		to_chat(user, "<span class='warning'>The linked account doesn't have the funds for that.</span>")
+		var/difference = amount_to_remove - registered_account.account_balance
+		to_chat(user, "<span class='warning'>The linked account needs [difference] more credit\s to perform that withdrawal.</span>")

```

Comment:
Consider using `registered_account.bank_card_talk(msg, TRUE)` here to make sure economy-related feedback stays consistent.

Reply:
IDK, i feel like there's no benefit to not receiving this info if you're deaf. Also, this is consistent in terms of feedback via `to_chats` for things regarding user input

Reply:
I was envisioning more of an ATM vibe (which the bank_card_talk format would fit better), but you could make a decent argument for either way.  It's probably fine.

Reply:
yeah true here would be good

Reply:
well alright